doc/source/cluster/cluster_under_construction/ray-clusters-on-kubernetes/user-guides/autoscaling.md

```diff
@@ -30,6 +30,64 @@ while the Ray application is running. On the other hand, if you pre-provision a
 number of Ray workers, all of the Ray workers can be started in parallel, potentially reducing your application's
 runtime.
 
+### Getting Started with Autoscaling
+
+First, follow the [quickstart guide](kuberay-quickstart) to create an autoscaling cluster. The commands to create the KubeRay operator and deploy an autoscaling cluster are summarized here:
+
+```bash
+# Optionally use kind to run the examples locally.
+# kind create cluster
+
+$ git clone https://github.com/ray-project/kuberay -b release-0.3
+# Create the KubeRay operator.
+$ kubectl create -k kuberay/ray-operator/config/default
+# Create an autoscaling Ray cluster.
+$ kubectl apply -f kuberay/ray-operator/config/samples/ray-cluster.autoscaler.yaml
+```
+
+Now, we can run a Ray program on the head pod that asks the autoscaler to scale the cluster to a total of 3 CPUs. The head and worker in our example cluster each have a capacity of 1 CPU, so the request should trigger upscaling of an additional worker pod.
+
+Note that in real-life scenarios, you will want to use larger Ray pods. In fact, it is advantageous to size each Ray pod to take up an entire Kubernetes node. See the [configuration guide](kuberay-config) for more details.
```

Comment:
Can we duplicate this pair of sentences in the quickstart?

It's sufficiently important and non-obvious that it wouldn't hurt spam five times on every page in the k8s section.